### DISCUSSION TITLE
Looking For Contributors!

### DISCUSSION THREAD

**Discussion by bitwalker** [bitwalker/timex#646] [General]
Hey everyone!

As you've probably been aware, maintenance has been a bit slow for some time, mostly because Timex more or less works as desired for my use cases, and apparently that is true for most users of the library based on the rate at which issues are reported. That said, there have been some longstanding bugs that I just recently took time to flush out and get fixed, culminating in the 3.7.x releases. While there may still be some kinks to be worked out there, it is a much better library as a result of those changes.

That said, even though I've made the time to do this work, I am utterly swamped with projects, and I want to make sure Timex continues to thrive. In order to make sure it does so, I wanted to reach out to the community and see if there are folks with ideas that are interested in actively participating in the project - both for general maintenance as well as adding functionality that is unlikely to be added to the standard library, or improves support for features within Timex (e.g. custom parsers/formatters, support for alternative timezone databases and calendars, etc.). While Timex is largely "complete", there are still plenty of corners to be covered with tests to ensure that there aren't latent bugs, and as the standard library provides more functionality, Timex can be refactored to focus more and more on the extras it provides.

If anyone is interested, this discussion is the place to let me know. I'm open to having multiple contributors, so don't worry if someone else has already expressed interest!

@ckhrysze since we haven't been in touch in a long while, would love to hear if you are still interested in maintaining the project; if not, no worries!

**Comment by drselump14** [comment]
Interested!

**Comment by corroded** [comment]
@bitwalker  I am interested in helping out. Disclaimer: my Elixir knowledge is kinda limited, but I do use it for work (as an Engineering Manager for a team that uses it). I'm happy to help out with releases etc. Currently the thing motivating me is the unreleased tzdata update and the missing tag for v3.7.11. Happy to do that for you and succeeding releases if needed :)
